tunisie/info.json

```diff
@@ -1,6 +1,5 @@
 {
   "app_name": "Encyclopédie de la Tunisie",
-  "version_name": "2018-07",
```

Comment:
why is the versionName getting removed, this is now an invalid json file?

Reply:
Well technically gradle could use the permalinks it just doesn't right now, it would be a bit tricky but I may be able to reuse the app's net code, it would probably be easier to just copy the classes to buildSrc folder.